js/components/accessibilityControls.js

```diff
@@ -3,33 +3,45 @@ var Utils = require('./utils');
 
 var AccessibilityControls = function (controller) {
   this.controller = controller;
+  this.allowed = true;
   this.state = {
     seekRate: 1,
-    lastKeyDownTime: 0
+    lastKeyDownTime: 0,
+    "keyMapsList": [
```

Comment:
We should try to keep style consistent. The rest of the file uses single quotes and also doesn't quote property names.

Reply:
\+